sstables_loader.cc

```diff
@@ -373,9 +373,8 @@ tablet_sstable_streamer::get_sstables_by_tablet_range(const shared_sstables& sst
         } else {
             sstables_partially_contained.push_back(sst);
         }
-        co_await coroutine::maybe_yield();
     }
-    co_return std::make_tuple(std::move(sstables_fully_contained), std::move(sstables_partially_contained));
+    return std::make_tuple(std::move(sstables_fully_contained), std::move(sstables_partially_contained));
```

Comment:
The question is how long can the sstables list grow, and whether this loop can stall?